### PR TITLE
Issue #3051 - New Simplified Jenkinsfile

### PR DESCRIPTION
**Pull request by joakime**
Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

### CODE REVIEW

**Comment by joakime** [comment]
I want to fix issue #2458 after this (removing jdk9 and jdk10 builds from CI)

**Comment by joakime** [comment]
Referencing Issue #3051 (typo in original commit)

File: Jenkinsfile [hunk 1]
```diff
@@ -1,19 +1,18 @@
 #!groovy
 
-// in case of change update method isMainBuild
-def jdks = ["jdk8","jdk9","jdk10","jdk11"]
+def mainJdk = "jdk8"
+def jdks = [mainJdk, "jdk9", "jdk10", "jdk11"]
```
Comment (sbordet) [inline Jenkinsfile]: jdk9 and jdk10 are not removed?

Comment (olamy) [reply Jenkinsfile]: not yet in jetty-9.4.x (maybe could make sense? I don't really have strong opinion)

Comment (joakime) [reply Jenkinsfile]: Removing jdk9 and jdk10 will be a different PR/commit (it's a different issue: #2458)

**Pull request merged**